docs/tutorials/mastering.md

```diff
@@ -1,60 +1,651 @@
-| Control Plane Kubernetes Cluster     | 4 | 8g | 40g | `management` |
-| User Plane Kubernetes Cluster        | 4 | 4g | 20g | `management`, `access`, `core` |
-| Juju Controller + Kubernetes Cluster | 2 | 8g | 40g | `management` |
-| gNB Simulator Kubernetes  Cluster    | 4 | 4g | 20g | `management`, `ran` |
+A machine running Ubuntu 22.04 with the following resources:
 
-
-## Sample Values
-
-For the purpose of this guide, the following values will be used.
+- At least one NIC with internet access
+- 8 cores
+- 24 GB RAM
+- 150 GiB disk  
 
 ### Networks
 
+The following IP networks will be used to connect and isolate the network functions.
+
 | Name | Subnet | Gateway IP |
 | ---- | ------ | ---------- |
 | `management` | 10.201.0.0/24 | 10.201.0.1 |
 | `access`     | 10.202.0.0/24 | 10.202.0.1 |
 | `core`       | 10.203.0.0/24 | 10.203.0.1 |
 | `ran`        | 10.204.0.0/24 | 10.204.0.1 |
 
-### IP Addresses
+On the host machine, create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/99-sdcore-networks.yaml`.
+Before creating the configuration of the network bridges, please make sure that:
+
+- mgmt-br route metric value is higher than your default route's metric
+- core-br metric value is higher than your mgmt-br route's metric
+
+Change the metrics of SD-Core routes which are indicated with comments below, relatively to your default route's metric if required. 
+
+```console
+cat << EOF | sudo tee /etc/netplan/99-sdcore-networks.yaml
+# /etc/netplan/99-sdcore-networks.yaml
+network:
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110 # Set the value higher than your default route's metric
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203 # Set the value higher than your mgmt-br route's metric
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+
+  version: 2
+EOF
+```
+
+Arrange the file permissions and apply the network configuration.
+
+```console
+sudo chmod 600 /etc/netplan/99-sdcore-networks.yaml
+sudo netplan apply
+```
+
+```{note}
+Applying new netplan configuration may produce warnings related to file permissions being too open. You may safely disregard them.
+```
+
+### Install and Configure LXD
+
+Install LXD:
+
+```console
+sudo snap install lxd
+```
+
+Create custom storage pool and LXD project:
+
+```console
+lxc storage create sdcore zfs size=150GiB
+lxc project create sdcore --config features.images=true --config features.networks=true --config features.networks.zones=true --config features.profiles=true --config features.storage.buckets=true --config features.storage.volumes=true
```

Comment:
Actually, the note about the LXD version should go under this line.
```note
When using LXD version lower than 5.9, the command above will produce the following error:
> Error: Invalid project configuration key "features.networks.zones"
Please upgrade your LXD to the latest version or remove unsupported config from the command.
```

Reply:
Done